src/gui/status_footer.cpp

```diff
@@ -220,17 +225,23 @@ void status_footer_t::repaint_heatbed() {
     wt_heatbed.SetTextColor(clr);
 }
 
+void status_footer_t::update_sheet_profile() {
+    sheet_current_name(text_z_profile, MAX_SHEET_NAME_LENGTH);
+    wt_z_profile.SetText(string_view_utf8::MakeCPUFLASH((const uint8_t *)text_z_profile));
+    wi_z_profile.SetIdRes(IDR_PNG_sheet_profile);
+}
+
 status_footer_t::status_footer_t(window_t *parent)
     : window_frame_t(parent, GuiDefaults::RectFooter)
     , wi_nozzle(this, Rect16(8, 270, 16, 16), IDR_PNG_status_icon_nozzle)
     , wi_heatbed(this, Rect16(128, 270, 20, 16), IDR_PNG_status_icon_heatbed)
     , wi_prnspeed(this, Rect16(10, 297, 16, 12), IDR_PNG_status_icon_prnspeed)
-    , wi_z_axis(this, Rect16(80, 297, 16, 16), IDR_PNG_status_icon_z_axis)
+    , wi_z_profile(this, Rect16(74, 297, 16, 16), IDR_NULL)
```

Comment:
`wi_z_profile(this, Rect16(74, 297, 16, 16), IDR_NULL)`
just draws black rectangle. Right?

Reply:
the correct icon is selected in the runtime and it depends on the number of calibrated sheets